### PR TITLE
Update apps.md to reflect new functions in /api/v1/apps endpoint.

### PR DESCRIPTION
**Pull request by ganghoonkim-okta**
Two new sections have been added to address the new functionality to the mentioned endpoint, which allows the callers to create App Wizard SAML and SWA apps through the API, rather than the UI. 
The example Application object has been updated to reflect the current model more closely. 
A new section has been added for `Attribute Statements Object`.

Code JIRA : [OKTA-83462](https://oktainc.atlassian.net/browse/OKTA-83462)
Doc JIRA : [OKTA-84290](https://oktainc.atlassian.net/browse/OKTA-84290)

Primary Reviewer: @mystiberry-okta 
Additional Reviewers: @bng-okta @karlmcguinness-okta @jpf

Gifs for visuals:
Attribute Statements Object
doc update_as

SWA
doc update_swa

SAML
doc update_saml

### CODE REVIEW

File: _source/docs/api/resources/apps.md [hunk 1]
```diff
@@ -1278,6 +1465,240 @@ curl -v -X POST \
...
+recipient             | The location where the app may present the SAML assertion                                                         | String                                               | FALSE    | FALSE |  [URL](http://tools.ietf.org/html/rfc3986)
+destination           | Identifies the location where the SAML response is intended to be sent inside of the SAML assertion               | String                                               | FALSE    | FALSE |  [URL](http://tools.ietf.org/html/rfc3986)
+audience              | Audience URI (SP Entity ID)                                                                                       | String                                               | FALSE    | FALSE |
+idpIssuer             | SAML Issuer ID                                                                                                    | String                                               | FALSE    | FALSE |
+subjectNameIdTemplate |                                                                                                                   | String                                               | TRUE     | FALSE |
```
Comment (ganghoonkim-okta) [inline _source/docs/api/resources/apps.md]: @bng-okta might need your help trying to describe this

File: _source/docs/api/resources/apps.md [hunk 2]
```diff
@@ -1278,6 +1465,240 @@ curl -v -X POST \
 }' "https://${org}.okta.com/api/v1/apps"
 ~~~
 
+#### Add Custom SAML Application
```
Comment (ganghoonkim-okta) [inline _source/docs/api/resources/apps.md]: @mystiberry-okta @bng-okta Was not sure if we used "App Wizard" publicly, so I used "Custom" instead.

**Comment by mystiberry-okta** [comment]
@ganghoonkim-okta @bng-okta this looks great, I'm ready to merge if my minor edits are okay. 
I'm fuzzy on the timing though, should this be available for 2016.26 or 2016.27 or...? Thanks!

**Comment by ganghoonkim-okta** [comment]
@mystiberry-okta right now we ran into some unexpected issues, and will probably end up merging in the code changes in 3 weeks, so this should be held off as well since we dont want to update the docs page with a functionality that isn't enabled yet. This will give us more chances to review it again and make changes without having to feel the pressure. 

fyi @bng-okta

**Comment by mystiberry-okta** [comment]
sounds good!

Mysti

On Jun 30, 2016, at 1:28 PM, ganghoonkim-okta <notifications@github.com<mailto:notifications@github.com>> wrote:


@mystiberry-okta<https://github.com/mystiberry-okta> right now we ran into some unexpected issues, and will probably end up merging in the code changes in 3 weeks, so this should be held off as well since we dont want to update the docs page with a functionality that isn't enabled yet. This will give us more chances to review it again and make changes without having to feel the pressure.

fyi @bng-okta<https://github.com/bng-okta>

**Pull request merged**